charts/policy-controller/README.md

```diff
@@ -54,7 +54,7 @@ The Helm chart for Policy  Controller
 | webhook.extraArgs | object | `{}` |  |
 | webhook.image.pullPolicy | string | `"IfNotPresent"` |  |
 | webhook.image.repository | string | `"ghcr.io/sigstore/policy-controller/policy-controller"` |  |
-| webhook.image.version | string | `"sha256:2b1c017535f6a0f672ec38279f3792ca1181555342a2deae53605e202afb9764"` | `"v0.4.0"` |
```

Comment:
please, keep the version here.

Reply:
the v0.4.0 or the sha?

Reply:
i believe is to keep the version in the `description` column